### ISSUE TITLE
ValueError: Unterminated string starting at: line 1 column 11 (char 10)

### ISSUE THREAD

**Issue by theartoflogic** [docker/compose#2528]
I'm running on OSX Yosemite v10.10.5

Here's some info about my environment:

```
> docker version
Client:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.5.1
 Git commit:   a34a1d5
 Built:        Sat Nov 21 00:48:57 UTC 2015
 OS/Arch:      darwin/amd64

Server:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.3
 Git commit:   a34a1d5
 Built:        Fri Nov 20 17:56:04 UTC 2015
 OS/Arch:      linux/amd64
```

```
> docker-compose version
docker-compose version: 1.4.0
docker-py version: 1.3.1
CPython version: 2.7.9
OpenSSL version: OpenSSL 1.0.1j 15 Oct 2014
```

```
> docker-machine inspect legacy
{
    "ConfigVersion": 1,
    "Driver": {
        "IPAddress": "192.168.99.109",
        "SSHUser": "docker",
        "SSHPort": 55988,
        "MachineName": "legacy",
        "CaCertPath": "/Users/sarahr/.docker/machine/certs/ca.pem",
        "PrivateKeyPath": "/Users/sarahr/.docker/machine/certs/ca-key.pem",
        "SwarmMaster": false,
        "SwarmHost": "tcp://0.0.0.0:3376",
        "SwarmDiscovery": "",
        "CPU": 1,
        "Memory": 1024,
        "DiskSize": 20000,
        "Boot2DockerURL": "",
        "Boot2DockerImportVM": "",
        "HostOnlyCIDR": "192.168.99.1/24"
    },
    "DriverName": "virtualbox",
    "HostOptions": {
        "Driver": "",
        "Memory": 0,
        "Disk": 0,
        "EngineOptions": {
            "ArbitraryFlags": [],
            "Dns": null,
            "GraphDir": "",
            "Env": [],
            "Ipv6": false,
            "InsecureRegistry": [],
            "Labels": [],
            "LogLevel": "",
            "StorageDriver": "",
            "SelinuxEnabled": false,
            "TlsCaCert": "",
            "TlsCert": "",
            "TlsKey": "",
            "TlsVerify": true,
            "RegistryMirror": [],
            "InstallURL": "https://get.docker.com"
        },
        "SwarmOptions": {
            "IsSwarm": false,
            "Address": "",
            "Discovery": "",
            "Master": false,
            "Host": "tcp://0.0.0.0:3376",
            "Image": "swarm:latest",
            "Strategy": "spread",
            "Heartbeat": 0,
            "Overcommit": 0,
            "TlsCaCert": "",
            "TlsCert": "",
            "TlsKey": "",
            "TlsVerify": false,
            "ArbitraryFlags": []
        },
        "AuthOptions": {
            "StorePath": "",
            "CaCertPath": "/Users/sarahr/.docker/machine/certs/ca.pem",
            "CaCertRemotePath": "",
            "ServerCertPath": "/Users/sarahr/.docker/machine/machines/legacy/server.pem",
            "ServerKeyPath": "/Users/sarahr/.docker/machine/machines/legacy/server-key.pem",
            "ClientKeyPath": "/Users/sarahr/.docker/machine/certs/key.pem",
            "ServerCertRemotePath": "",
            "ServerKeyRemotePath": "",
            "PrivateKeyPath": "/Users/sarahr/.docker/machine/certs/ca-key.pem",
            "ClientCertPath": "/Users/sarahr/.docker/machine/certs/cert.pem"
        }
    },
    "StorePath": "/Users/sarahr/.docker/machine/machines/legacy"
}
```

I'm using `docker-compose build <service>` and during the build process I'm getting the following error:

```
Traceback (most recent call last):
  File "<string>", line 3, in <module>
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.main", line 39, in main
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.docopt_command", line 21, in sys_dispatch
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 27, in dispatch
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.docopt_command", line 24, in dispatch
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 59, in perform_command
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.main", line 135, in build
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.project", line 233, in build
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.service", line 714, in build
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.progress_stream", line 18, in stream_output
  File "/compose/build/docker-compose/out00-PYZ.pyz/json", line 338, in loads
  File "/compose/build/docker-compose/out00-PYZ.pyz/json.decoder", line 366, in decode
  File "/compose/build/docker-compose/out00-PYZ.pyz/json.decoder", line 382, in raw_decode
ValueError: Unterminated string starting at: line 1 column 11 (char 10)
```

It's happening consistently when a `RUN npm install` command in my Dockerfile is about to finish up. However, it only happens when certain npm modules are installed. When I remove certain modules it runs through fine. One module I've been able to reproduce the issue with consistently is `babel-preset-es2015`. Whenever I have that module in my package.json file the `docker-compose build` command fails with the above error.

Does anyone have any idea what the issue could be?

**Comment by dnephin** [comment]
I think this is a bug in older `docker-compose` versions that was fixed in 1.5.x. Could you try with 1.5.2 ?

**Comment by dnephin** [comment]
No response from OP, closing.
